database_interaction/database_insert.py

```diff
@@ -1,7 +1,10 @@
 from database_interaction import database
+from database_interaction import time_added, time_update
 
 
 def insert_mydata(my_dict: dict):
+    my_dict["time_created"] = time_added.song_time_added()
+    my_dict["time_updated"] = time_update.my_update_time()
```

Comment:
support allowing the user to insert the date by himself and move this to the scripts folder